lib/anyOf.js

```diff
@@ -0,0 +1,38 @@
+
+const schema = require('./schema')
+
+module.exports = compileAnyOf
+
+function compileAnyOf (gen, prop, rawSchema, genany) {
+  gen('const codeAtPtr = s.charCodeAt(ptr)')
```

Comment:
use gen.sym('codeAtPtr') to generate a unique variable name based on that :) - that makes sure the var name wont clash when where is multiple nested anyOfs